packages/mask/src/plugins/Avatar/utils/index.ts

```diff
@@ -83,9 +83,23 @@ export const sortPersonaBindings = (a: NextIDPersonaBindings, b: NextIDPersonaBi
     return 1
 }
 
+export async function getNextIDBindings(userId: string) {
+    const platform = activatedSocialNetworkUI.configuration.nextIDConfig?.platform as NextIDPlatform
+
+    const nextIDPersonaBindings: NextIDPersonaBindings[] = []
+    let page = 0
+    do {
+        const personaBindings = await NextIDProof.queryExistedBindingByPlatform(platform, userId.toLowerCase(), page)
```

Comment:
We could add `queryAllExistedBindingByPlatform` to `NextIDProof`. It could be needed by others.